src/components/projectGallery/ProjectCard.tsx

```diff
@@ -44,7 +56,25 @@ const ProjectCard: React.FC<Props> = props => {
         </p>
       )}
       {tags.map((tag: string) => (
-        <Tag>{tag}</Tag>
+        <>
+          <Flex>
+            <Tag>{tag}</Tag>
+            <Text fontSize="xs" color="blue.500" _hover={{ cursor: "pointer", textDecoration: "underline" }} onClick={() => handleComponentClick(tag)}>
+              What is this?
+            </Text>
+          </Flex>
```

Comment:
I think we should remove the "What is this" from this page. There are so many texts that it makes it distracting. I think having the explanation just on the admin page is enough and we don't need anything here.
<img width="1471" alt="Screenshot 2023-07-31 at 1 47 30 PM" src="https://github.com/HackGT/timber/assets/8273038/bbf0d190-a9d8-4b63-b0b4-507f628ff8c7">